packages/app/src/explorer/utils.ts

```diff
@@ -26,3 +26,22 @@ export function needsNxBadge(
 
   return false;
 }
+
+export function getActiveButtonIndex(
+  element: Element | null
+): [number, HTMLButtonElement[]] | undefined {
+  if (!element) {
+    return undefined;
+  }
+
+  const activeButton = document.activeElement;
+
+  if (!activeButton || !(activeButton instanceof HTMLButtonElement)) {
```

Comment:
This probably needs to be a bit stricter, since there are other buttons on the UI. Perhaps `element.contains(activeButton)`, or you could get the list of buttons first and see if it's inside. Same same.